osquery/main/main.cpp

```diff
@@ -140,6 +140,7 @@ int startShell(osquery::Initializer& runner, int argc, char* argv[]) {
     retcode = osquery::launchIntoShell(argc, argv);
     // Finally shutdown.
     runner.requestShutdown();
```

Comment:
Any reason why we don't need to call `runner.requestShutdown()` on the `startDaemon()` version?